### PR TITLE
Dashboard Scene: Chore - Add missing unit test to PanelDataQueryTab

### PR DESCRIPTION
**Pull request by axelavargas**
Added missing unit test from PR: https://github.com/grafana/grafana/pull/86883

### CODE REVIEW

**Review by Sergej-Vlasov — APPROVED** [review]
LGTM! 👍

File: public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx [hunk 1]
```diff
@@ -76,7 +76,7 @@ export class VizPanelManager extends SceneObjectBase<VizPanelManagerState> {
   public static createFor(sourcePanel: VizPanel) {
     let repeatOptions: Pick<VizPanelManagerState, 'repeat' | 'repeatDirection' | 'maxPerRow'> = {};
 
-    const gridItem = sourcePanel.parent instanceof LibraryVizPanel ? sourcePanel.parent.parent : sourcePanel.parent;
+    const gridItem = sourcePanel?.parent instanceof LibraryVizPanel ? sourcePanel.parent.parent : sourcePanel?.parent;
```
Comment (Sergej-Vlasov) [inline public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx:79]: @axelavargas is this change intended? Seems like these test work without this change?

Comment (axelavargas) [reply public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx:79]: good catch, not intended :)

**Review by ivanortegaalba — APPROVED** [review]

File: public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx (hunk 1 above)
Comment (axelavargas) [inline public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx:79]:
```suggestion
    const gridItem = sourcePanel.parent instanceof LibraryVizPanel ? sourcePanel.parent.parent : sourcePanel.parent;
```
